src/migration/1672583245776-SourceImageDefault.ts

```diff
@@ -7,7 +7,7 @@ export class SourceImageDefault1672583245776 implements MigrationInterface {
     await queryRunner.query(`ALTER TABLE "source"
       ALTER COLUMN "image" SET NOT NULL`);
     await queryRunner.query(`ALTER TABLE "source"
-      ALTER COLUMN "image" SET DEFAULT 'https://daily-now-res.cloudinary.com/image/upload/v1672041320/squads/squad_placeholder.jpg'`);
+      ALTER COLUMN "image" SET DEFAULT 'https://daily-now-res.cloudinary.com/image/upload/s--LrHsyt2T--/f_auto/v1692632054/squad_placeholder_sfwkmj'`);
```

Comment:
Shouldn't it be a new migration, instead of altering an already existing migration?

Reply:
Yeah it should be a new migration, this won't trigger

Reply:
I know it wont trigger, I just thought there is no point in updating defaults with current squads. 

I'll add the new one if needed 👍

Reply:
Ah yes, but we want the new default to be something correct?
So if squads get created without image it needs to default to this new one.

Reply:
Yes, mine oversight, will take care od it.